tools/wptserve/wptserve/handlers.py

```diff
@@ -273,7 +273,7 @@ def get_range_data(self, f, byte_range):
         return f.read(byte_range.upper - byte_range.lower)
 
 
-file_handler = FileHandler()
+file_handler = FileHandler()  # type: ignore
```

Comment:
Would it break anything to specify the class name here rather than using `# type: ignore` on some of these, e.g. `# type: FileHandler`?

Reply:
Whether removing this annotation or adding `# type: FileHandler`, the error is 'Call to untyped function "FileHandler" in typed context'. So the problem isn't that the return type is unknown, but that the `__init__` method of `FileHandler` isn't typed. Adding types isn't super hard, but with a PR like this just want to enable optional typing, and then one can add types over time. If `FileHandler` is annotated, this `type: ignore` annotation will then result in a 'Unused "type: ignore" comment' error, so it won't be left around.